enterprise/utils.py

```diff
@@ -143,6 +143,34 @@ def get_idp_choices():
     return None
 
 
+def get_provider_login_url(request, provider_id):
+    """
+    Return the given provider's login URL.
+    """
+    try:
+        from third_party_auth import pipeline
+    except ImportError as exception:
+        LOGGER.warning("Could not import Pipeline from third_party_auth")
+        LOGGER.warning(exception)
```

Comment:
it is not recommended to use same logger in adjacent lines, we can use single logger to print a complete message.
If you are intended to print stack_trace than you should use either `exception` logger or pass param `exc_info=True`